tests/Aspire.Hosting.Tests/Schema/SchemaTests.cs

```diff
@@ -0,0 +1,296 @@
+                    }
+                },
+
+                { "BasicProject", (IDistributedApplicationBuilder builder) =>
+                    {
+                        builder.AddProject<Projects.ServiceA>("project");
+                    }
+                },
+
+                { "BasicExecutable", (IDistributedApplicationBuilder builder) =>
+                    {
+                        builder.AddExecutable("executable", "hellworld", "foo", "arg1", "arg2");
+                    }
+                },
+
+                { "AwsStack", (IDistributedApplicationBuilder builder) =>
+                    {
+                        var awsSdkConfig = builder.AddAWSSDKConfig()
+                                                  .WithRegion(RegionEndpoint.USWest2)
+                                                  .WithProfile("test-profile");
+
+                        builder.AddAWSCloudFormationStack("ExistingStack")
+                               .WithReference(awsSdkConfig);
+                    }
+                },
+
+                { "AwsTemplate", (IDistributedApplicationBuilder builder) =>
+                    {
+                        var awsSdkConfig = builder.AddAWSSDKConfig()
+                                                  .WithRegion(RegionEndpoint.USWest2)
+                                                  .WithProfile("test-profile");
+
+                        builder.AddAWSCloudFormationTemplate("TemplateStack", "nonexistenttemplate")
+                               .WithReference(awsSdkConfig);
+                    }
+                },
+
+                { "DaprWithComponents", (IDistributedApplicationBuilder builder) =>
+                    {
+                        var dapr = builder.AddDapr(dopts =>
+                        {
+                            // Just to avoid dynamic discovery which will throw.
+                            dopts.DaprPath = "notrealpath";
+                        });
+                        var state = dapr.AddDaprStateStore("daprstate");
+                        var pubsub = dapr.AddDaprPubSub("daprpubsub");
+
+                        builder.AddProject<Projects.ServiceA>("project")
+                               .WithDaprSidecar()
+                               .WithReference(state)
+                               .WithReference(pubsub);
+                    }
+                }
+            };
+
+            return data;
+        }
+    }
+
+    private static JSchema? s_schema;
+
+    private static async Task<JSchema> GetSchemaAsync()
+    {
+        if (s_schema == null)
+        {
+            var relativePath = Path.Combine("Schema", "aspire-8.0.json");
+            var schemaPath = Path.GetFullPath(relativePath);
+            var schemaText = await File.ReadAllTextAsync(schemaPath);
+            s_schema = JSchema.Parse(schemaText);
+        }
+
+        return s_schema;
+    }
+
+    [SkipOnHelixTheory]
```

Comment:
We shouldn't need to skip these tests on Helix. We are doing the same thing in the Components tests, and we can make it work by doing this:

https://github.com/dotnet/aspire/blob/b0e3578fc56cd45cd41897a66a446f2c341c9726/tests/Aspire.Azure.AI.OpenAI.Tests/Aspire.Azure.AI.OpenAI.Tests.csproj#L8